provider-gui/types/ContractOverview.ts

```diff
@@ -0,0 +1,15 @@
+export interface ContractOverview {
+    contractAddress: string
+    network: 'development' | 'rococo'
```

Comment:
should be populated from some config somewhere, same as above

Reply:
Ticket in